### ISSUE TITLE
Plot loss during training

### ISSUE THREAD

**Issue by chenzhao2016** [MouseLand/cellpose#594]
Dear all, 

I want to fine-tune parameters for Cellpose training. I wonder if the information of the loss function during each Epoch round is saved somewhere, such as a log file including: 
2022-01-05 21:36:32,809 [INFO] Epoch 190, Time 43354.3s, Loss 0.6758, Loss Test 0.3793, LR 0.2000
2022-01-05 22:14:13,626 [INFO] Epoch 200, Time 45615.1s, Loss 0.6738, Loss Test 0.3603, LR 0.2000

Has anyone implemented TensorBoard to monitor training?

**Comment by carsen-stringer** [comment]
in `$HOME/.cellpose/run.log`, this info is saved, but there is currently no tensorboard support
